SPECS/golang/golang.spec

```diff
@@ -14,7 +14,7 @@
 Summary:        Go
 Name:           golang
 Version:        1.20.7
```

Comment:
`go 1.20.10` was released today.
IMO, we should probably take that instead of a patch.

Reply:
In general - yes. In case of this PR - no.:) We'll bump the version in a separate change. This is a high-priority CVE fix, so we want to minimize any potential for build failures.